pip/vcs/subversion.py

```diff
@@ -263,6 +265,14 @@ def check_version(self, dest, rev_options):
         """Always assume the versions don't match"""
         return False
 
+    @staticmethod
+    def remove_auth_from_url(url):
+        # Return a copy of url with 'username:password@' stripped.
+        # username/pass params are passed to subversion through flags & not recognized in
+        #  the url.
+        url = re.sub('://.*?@', '://', url, count=1)
```

Comment:
It looks like this regex would not work well with `svn+http://server/svn/project/file@1234` and give `svn+http://1234`...